src/components/shared/sidemenu/SideMenu.jsx

```diff
@@ -1,31 +1,45 @@
 import react from "react";
+import { NavLink } from "react-router-dom";
 import { ImHome } from "react-icons/im";
 import { MdExplore, MdWatchLater } from "react-icons/md";
-import { RiPlayList2Fill, RiSettings4Fill,RiHistoryLine } from "react-icons/ri";
+import {
+  RiPlayList2Fill,
+  RiSettings4Fill,
+  RiHistoryLine,
+} from "react-icons/ri";
 import { AiFillLike } from "react-icons/ai";
 
-
 import "./sidemenu.css";
 
 const SideMenu = () => {
   const menuOptions = [
-    { name: "Home", icon: <ImHome /> },
-    { name: "Explore", icon: <MdExplore /> },
-    { name: "Playlist", icon: <RiPlayList2Fill /> },
-    { name: "Watch Later", icon: <MdWatchLater /> },
-    { name: "Liked", icon: <AiFillLike /> },
-    { name: "History", icon: <RiHistoryLine /> },
-    { name: "Settings", icon: <RiSettings4Fill /> },
+    { name: "Home", icon: <ImHome />, path: "/home" },
+    { name: "Explore", icon: <MdExplore />, path: "/explore" },
+    { name: "Playlist", icon: <RiPlayList2Fill />, path: "/playlist" },
+    { name: "Watch Later", icon: <MdWatchLater />, path: "/watchlater" },
+    { name: "Liked", icon: <AiFillLike />, path: "/liked" },
+    { name: "History", icon: <RiHistoryLine />, path: "/history" },
+    { name: "Settings", icon: <RiSettings4Fill />, path: "/mockman" },
   ];
+
+  const activeStyle = {
+    color: "red",
+    borderBottom:"0px"
+  };
+
   return (
-    <aside className="sidemenu-container">
+    <aside className="sidemenu-container sticky-sidemenu">
       <ul>
-        {menuOptions.map(({ name, icon }) => (
-          <li key={name} className="menu-item">
-            {" "}
-            <div className="menu-icon">{icon}</div>{" "}
-            <div className="menu-title">{name}</div>{" "}
-          </li>
+        {menuOptions.map(({ name, icon, path }) => (
+          <NavLink
+            to={path}
+            key={name}
+            className="menu-item"
+            style={({ isActive }) => isActive ?activeStyle:undefined}
```

Comment:
You can just use the `active` className which will be added when you use `NavLink`

Reply:
ok